### PR TITLE
[CFM-904] Adding 'join to group' links on the CTA block for visitors.

### PR DESCRIPTION
**Pull request by ordavidil**
https://issuetracker.amplexor.com/jira/browse/CFM-904

### CODE REVIEW

**Comment by ordavidil** [comment]
selection_217
selection_216

**Comment by ordavidil** [comment]
Removed the log in link:

selection_218

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module [hunk 1]
```diff
@@ -421,3 +421,61 @@ function c4m_user_og_c4m_helper_entity_metrics_info() {
...
+      'button_icon' => 'fa-unlock-alt',
+    ),
+  );
+
+  $membership_request_value = c4m_og_get_group_membership_request($group);
+
+  $variables = array(
+    'button_label' => $types[$membership_request_value]['button_label'],
+    'button_icon' => $types[$membership_request_value]['button_icon'],
+    'url' => url('/user'),
```
Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: Links to <group_purl>/user, meaning the group navigation is still visible.

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: was possible, but already merged.

**Pull request merged**